backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphlAdminFieldFactory.java

```diff
@@ -32,6 +33,11 @@ private GraphlAdminFieldFactory() {
                   .name(SETTINGS)
                   .type(GraphQLList.list(outputSettingsType))
                   .build())
+          .field(
+              GraphQLFieldDefinition.newFieldDefinition()
+                  .name(ROLES)
```

Comment:
is this the roles of current user? I would have expected a nested field.

Probably {_users{name,roles{name}}